site/sfguides/src/native-app-chairlift/native-app-chairlift.md

```diff
@@ -0,0 +1,641 @@
+author: Gilberto Hernandez
+id: native-app-chairlift
+summary: This Snowflake Native App demonstrates how a chairlift manufacturer can build a native app to provide chairlift customers with analytics on purchased chairlifts by using sensor data collected from the chairlifts owned by the customer.
+categories: Getting-Started
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Data Science, Data Engineering, Apps 
+
+# Build a Snowflake Native App to Analyze Chairlift Sensor Data
+<!-- ------------------------ -->
+## Introduction
+Duration: 1
+
+In this Quickstart, you'll build a Snowflake Native Application that can analyze sensor data from chairlifts at different locations at a ski resort. Let's explore the scenario from the perspective of the application provider and an application consumer.
```

Comment:
Perhaps you can mention that this quickstart is limited to a single-account installation, and that things like listing / versions / release directives are outside of the scope.